lib/ClangImporter/ImportDecl.cpp

```diff
@@ -3642,9 +3642,34 @@ namespace {
 
       result->setHasUnreferenceableStorage(hasUnreferenceableStorage);
 
-      if (cxxRecordDecl)
+      if (cxxRecordDecl) {
         result->setIsCxxNonTrivial(!cxxRecordDecl->isTriviallyCopyable());
 
+        SmallVector<SubscriptDecl *, 1> subscripts;
+        for (auto &subscriptInfo : Impl.cxxSubscripts) {
+          auto declAndParameterType = subscriptInfo.first;
+          if (declAndParameterType.first != result)
+            continue;
+
+          auto getterAndSetter = subscriptInfo.second;
+          auto subscript = makeSubscript(getterAndSetter.first,
+                                         getterAndSetter.second);
+          subscripts.push_back(subscript);
+        }
+        // Make sure the subscripts are added in a stable order.
+        auto &sourceManager = Impl.getClangASTContext().getSourceManager();
+        std::sort(subscripts.begin(), subscripts.end(),
```

Comment:
Nit: you can use the `llvm::sort` overload and then just do `llvm::sort(subscripts, [](){ ... })`